lib/features/digital_guide_view/data/readme.md

```diff
@@ -0,0 +1,14 @@
+# Requirements
+* All HTTP requests must include the authorization token (DIGITAL_GUIDE_URL)
+
+# Tips
+* To obtain the authorization token, username, or password for API login, please contact the development team
+* The complete list of endpoints is available after logging under [this](https://przewodnik.pwr.edu.pl/swagger/) link
+
+# Used endpoints
+1) Building data
+  * DIGITAL_GUIDE_URL/buildings/{id}
+2) Building image
+  * DIGITAL_GUIDE_URL/images/{id}
+3) Surroundings data
+  * DIGITAL_GUIDE_URL/surroundings/{id}
```

Comment:
I know Mikołaj just told you to remove repositories names here, but if this docs are meant to have any meaning, they should somehow tell us where in the code we're using this endpoints. But we can use our so called "subfeatures" names that we'll name our subcatalogs too

Reply:
if we have subfeatures catalogs like "general_info" or "surroundings", we can mention them here. Cause I think every of such "subfeatures" is gonna be like 1 or 2 endpoints.